etc/org.opencastproject.index.service.resources.list.provider.ContributorsListProvider.cfg

```diff
@@ -0,0 +1,14 @@
+# A comma separated list of user provider names to exclude the users from the result set.
+# The available user provider names depends on your system configuration.
+# The may be incomplete list of provider names:
+#   - opencast: Opencast managed users
+#   - system: Opencast internal users e.g. digest user
+#   - matterhorn-reference: referenced users e.g. LTI user references
+#   - ldap: LDAP user directory
+#   - canvas: Canvas user provider
+#   - moodle: Moodle referenced users
+#   - sakai: Sakai referenced users
+# A special value to match all available providers is: *. Prefere to use this value if you want to exclude all user
+# providers. It will improve performance a lot.
```

Comment:
```suggestion
# A special value to match all available providers is `*`.
# This can improve the performance quite a bit if you do not need the users.
```

Reply:
suggestion committed